packages/mask/.webpack/config.ts

```diff
@@ -152,6 +152,12 @@ export function createConfiguration(rawFlags: BuildFlags): Configuration {
                         },
                     },
                 },
+                {
+                    test: /\.m?js/,
+                    resolve: {
+                        fullySpecified: false,
```

Comment:
Please limit this rule to the broken package only. Do not apply it to the whole package scope.

Reply:
OK